From 0fc626c1562c8f3c67a07c243707597158255b77 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 22 Oct 2026 13:10:14 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh rsmarples.gpg from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 dhcpcd.spec   |   7 +++
 rsmarples.gpg | 142 ++++++++++++++++++++++++++++++++------------------
 2 files changed, 99 insertions(+), 50 deletions(-)

diff --git a/dhcpcd.spec b/dhcpcd.spec
index 5b803d4..af27dfc 100644
--- a/dhcpcd.spec
+++ b/dhcpcd.spec
@@ -31,6 +31,9 @@ BuildRequires: systemd-devel
 BuildRequires: ypbind
 %endif
 BuildRequires: make
+%if 0%{?fedora} || 0%{?rhel} >= 11
+BuildRequires: openpgpverify
+%endif
 %if 0%{?fedora} || 0%{?rhel} > 8
 BuildRequires: gnupg2
 %endif
@@ -42,8 +45,12 @@ through NDP, DHCPv4 and DHCPv6 protocols.
 
 %prep
 %if 0%{?fedora} || 0%{?rhel} > 8
+%if 0%{?fedora} || 0%{?rhel} >= 11
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%else
 %{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %endif
+%endif
 %autosetup -p1
 
 %build
diff --git a/rsmarples.gpg b/rsmarples.gpg
index ea32ae7..f868929 100644
--- a/rsmarples.gpg
+++ b/rsmarples.gpg
@@ -1,61 +1,103 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQENBEkrCoIBCACyCfZ4qEaFPi33OT1EYDziIOb5KnvW3E9zP7O8oHGmhzi+v11X
+xsBNBEkrCoIBCACyCfZ4qEaFPi33OT1EYDziIOb5KnvW3E9zP7O8oHGmhzi+v11X
 AsShyiQCFaKD3QgQ/Ra5TV6mjfjV2fPjSXp6wVy9ojvmIFpvIgu1AQJIqkYl0jpV
 JFNT1n9RcI3RkZdYYbxkncmp3M8NMhWo0Txpuw6dx2c7gLeQp0PQiwJUPiLS1rUw
 KQLFGnPfvMu4koCoxeyYydijR8XuMmEk3KkYxqDKw81IljwhvOiJa9AVik01GMnE
 C+9xQcrdaa5+0yqPt3n/9WrGk+yC2oLiSjGgJHNB7niU716gf5HVUYrqEdiuVw4+
-scSjEdCOmRoWcq6D84EIkoXXX0gurSLcHh7xABEBAAG0HlJveSBNYXJwbGVzIDxy
-b3lAbWFycGxlcy5uYW1lPokBNgQTAQIAIAUCSSsKggIbAwYLCQgHAwIEFQIIAwQW
-AgMBAh4BAheAAAoJEFl/l+qa1FVJurUH/0+FeruUnnv/rs2aOkXlO6QiDUCyjxE0
-9fkS6gmKvsZ6zk4Li+3c0xTbt8+qnI3rtNxtlqAiqyZqebBTMjdG2ToXrFDjXaYq
-3OTIlUsunXVOmz37LOwFsff4TTt4HcbhcMeAXNI2babdJc3NeNNmqPkoTJ9BTE0l
-5gswITv5X0NwrOKf5GQCN4yu8COC6f8bAvbBjKTiRaOimVyZWUDcbomN+U5/WZJ1
-z6AVghwriVcGhj4oZAY5aX9QPxnf0flP/oOLxXOZnUO9UVApE+wlHf4SFzbcPWEd
-bM9sq3+EBo+h++8xxSyzZQrhkY0Do79oagLO3A+QIPkjBx8B7MOXKOSJARwEEwEC
-AAYFAkksWu4ACgkQX3c5MRWyLZXRQgf8CvuuKEJNAo9LSfhjwuG5GmvyX9MaygW4
-ukpZKJm86C7NMn3KobdXFmIR+mt4SpvTkl9k1iUlA9BJY4cTEs5kw8hak22qYt37
-Hp2O+NAOq5KpohUjmL9DKQeGpUmsCMkk/sTNXxUh2KL/OnUHRhQJc9Z3r9V65kfX
-Z9e57Rrydrth2DEIZmvITV7Fx0McQcZSxONaTDZVThdl93f9LFS4H113CDf6tpn+
-iFFMjp+NjVPNr6pOTjE9E7/H2PzmMth2VUI54vWFQul/MHCe2OYmRaCNCtq48rgs
-/N6+0w1r0J/lZu8n04l1NaHlttN3bDW477hrSq4HWsaa8LnNPjocaokBOQQTAQIA
-IwIbAwYLCQgHAwIEFQIIAwQWAgMBAh4BAheABQJJPl+PAhkBAAoJEFl/l+qa1FVJ
-hBMIAKgNBVUm5sOuFBEx+7rRP1dkU6vFZio0sIWgKtj22TYqqmqfK/0zOMf3ztef
-kf1F63H2kuwB9r0q+vThzGaremDK5m2ewFxFUOm30pj1T/6M2s7Edz5Y8glrIDSc
-RVvpajeCKSWg8hnrxyS053OddrA+vWEWfSQPE3OVfeXvv6gtf64ttA7Z/hWEyHlO
-ftZJwHr4RHMO973nnkpffT7tFNBKlI9q/hbBLt8EDXtJ7iUBwwL//8GP0e1NIMtF
-SlYvDDATyd0mUn1CxznVc7oszZ/Txf5jvEZTgWQNWprqvqCVQnTSS3fIJmZft0XE
-yWlzfYg7YqU8a3kXqZBi7CVds6e0JVJveSBNYXJwbGVzIChOZXRCU0QpIDxyb3lA
-TmV0QlNELm9yZz6JARwEEwECAAYFAkk/+kgACgkQX3c5MRWyLZXtJwf+MUN0DsV7
-DmUFOPNOTHmiwcsdDJoKw6Ay2PbJZcrZuQZo0snkXWnKH4YAKoFdppBlRUvOiz8O
-Q8Z1ySRSy4NuVmFHkuOchg0iHy+g3hCwxU1HVMXUTixxsuab9EIJ6VZVCsbQR6l6
-yW10tZsGX1v+uTJ6/hgTojB8uSftVIKzIGxqZrLUKV+lU7BnQB7C1bqYBn9Rj+S3
-3l99M2R1V1QMjXM+VXdO8De15gb9q8oYGAfnHry6b3oL6/yW5hyERNaBHrkk33XX
-YiyFG9+JamcWfgEtZdw9DGvJqccZljUawWPmB3ILYIt9VZAbxK+23flZB5NTf/KP
-cmMrqRgB38THsYkBNgQTAQIAIAUCST5fbwIbAwYLCQgHAwIEFQIIAwQWAgMBAh4B
+scSjEdCOmRoWcq6D84EIkoXXX0gurSLcHh7xABEBAAHNJVJveSBNYXJwbGVzIChO
+ZXRCU0QpIDxyb3lATmV0QlNELm9yZz7CwJEEEwEIADsWIQSnhe0nVZVdnpPqWfZZ
+f5fqmtRVSQUCZEeIlAIbAwULCQgHAgIiAgYVCgkICwIEFgIDAQIeBwIXgAAKCRBZ
+f5fqmtRVSQraCACDTCMQDf8n8RqyPMMOniAV5qpwvIndZ32jo3vIESPRCfD4zpxE
+QJqk8fDOyQZ7drLVHdQ251OoY7WD4cuY9+410K7pl/bA3pbHs2rw0wkRjthX4muc
+Vo3Iv3zMuAXYfwAFLxkGlo24czKMiCvJRXXvQOqM+VBN/f1irwq7dlOnxn4I96ek
+Cj5zl8iF/7mTr/f14WM7v6bxQtA/K3WkOzmitzkR5/mdBOLIJi2UpW78C7JFgOWb
+WsbnziR1sSuB0e83S6HpTqkIUNxUUOkZh7eKJIKqbDlF0BRjV3CwrJyUXrBg92kC
+EMd5Bqhi4/Xuw9tSvm9UJkiMmASDkDbq9YaUwsCRBBMBCAA7FiEEp4XtJ1WVXZ6T
+6ln2WX+X6prUVUkFAmRGniYCGwMFCwkIBwICIgIGFQoJCAsCBBYCAwECHgcCF4AA
+CgkQWX+X6prUVUkZ3Af/SM3JskROKhpEIfNKr9R7EJGcVCSHb2eVgje+Z92iDTzK
+OkjimRzUbsCDBPZyMwx7Wj1qrRPAXDkKB9y1uTucwa527AIVaTPH+w0iT/AxZhlp
+H47R8y0g0JTt+6oFP/uHL+4Yd82WU5W9QGP9UZoTjxMdvKX9XkhwUrBWYMh5snTH
+Ixh1revPprP8P5Ntg7+xolT8RhVhcg9H4j9DYWTK/rZJ0DUHgRwYrVLR0WkrlTcM
+6iCCLKxseaBAPsMoY76mq3lVrqHdeLeKJCslObZvn7ph7drPgJ2r3dMVEXCliNcM
+kO6jdewGVPStoqvKmXsc9dy3pSbcu2/Mw/mt8eECs8LAeAQTAQIAIgUCS4AaSQIb
+AwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQWX+X6prUVUkecwf/bMfGlZ01
+x8wKQ69Sj1lUaNLYrnt8oIhvYQfCHJA+oVxjg60Vg0IkLvmziKZjnKbbg8J4XHFc
+bwYsyqBW0rkFuufHRQwjrzHpTM/7Zqn5IEMUTsVq50IBt1fjwAGSsd0sDtXMH4GT
+8BumoQgrPzqjw0VUV0DEwt3Cst1OMYbd+KBsYQ6oEiUWiDlz72KOfy6+FFhbd0hv
+pviFDqVo5+ZnAoBOFWHHcOT1EEf0OQd7uMPz48nTnN0KABrRzHvDsL94xyk01qQ8
+sBKmDMISZUKhuCm6gRxleYS/yEllhDdvPMoQgeHNA13kByTuvuvkysWykPrLlDjX
+0PJsvc2JT2Bx2MLAdgQTAQIAIAUCST5fbwIbAwYLCQgHAwIEFQIIAwQWAgMBAh4B
 AheAAAoJEFl/l+qa1FVJRSEIAINSIaIn+P1xbaRHYGHsZvp5naorkabdHJNm7Ylf
 iN50XONCkR8Lw2pYwjQKSYEeuIzJPHJRly/QkmeNWQlVzlFkYI/kFy/UpdWNqkCH
 v1fpq5ZdvSPemRiGiRqFPlLAHObgo0tbI46tfOr9eQrAqRbNGVgguqWE5e37LnwS
 CO5YYKm64pD4ObAcYgr3afBbftnMI/GHrI3IBzLBOGgpjNXP9trSr9e6rOMxFX3M
 Kp5JBJ1zcVeQewGdLbq2aimCqmT75P19GBrpg4gPAVo06WGDi7csmfo9ffWFnpet
-Ut+cDCM7zD5tcG/JTWyvw+b8ctXoceFtJFkkjHaZGBjkHpKJATgEEwECACIFAkuA
-GkkCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEFl/l+qa1FVJHnMH/2zH
-xpWdNcfMCkOvUo9ZVGjS2K57fKCIb2EHwhyQPqFcY4OtFYNCJC75s4imY5ym24PC
-eFxxXG8GLMqgVtK5Bbrnx0UMI68x6UzP+2ap+SBDFE7FaudCAbdX48ABkrHdLA7V
-zB+Bk/AbpqEIKz86o8NFVFdAxMLdwrLdTjGG3figbGEOqBIlFog5c+9ijn8uvhRY
-W3dIb6b4hQ6laOfmZwKAThVhx3Dk9RBH9DkHe7jD8+PJ05zdCgAa0cx7w7C/eMcp
-NNakPLASpgzCEmVCobgpuoEcZXmEv8hJZYQ3bzzKEIHhzQNd5Ack7r7r5MrFspD6
-y5Q419DybL3NiU9gcdi5AQ0ESSsK+gEIAMOYyNnU8QvuP68M3ODgzy3NgqwUViZC
-TXKiaJhpCgQWTvVYHFMN/DSCBKIWjpJnB11rVlzeRRO6sjOBl9wiIOCcvv20Soc2
-sRdysTH7K4stgLFIFXIqT9QxGeaJwe+loSwjtbrCZW7G90H4DMfhM/VbEZjv8174
-34XNIAd3fn2sykOnwm9B8vMdy8jk1/vj+/lmU/YYFkHLKausbkr+FHRaBLJyBBin
-/Po5D953BaSe/ATN2IGHSfPo0P4hqJ/G8Tg8QTeBysDk1UpxePY4Sas/rlkmyi/4
-ssn65BwsUe9W4BxOjX4XO5Pkg368CVo8/rk13uGqzqFFFkShj1BaRtEAEQEAAYkB
-HwQYAQIACQUCSSsK+gIbDAAKCRBZf5fqmtRVSU5lB/9NdyONWDwonoiRs7zNV2bf
-4+aUSS6G/koWHEuAlTRANWF2mzU52YrmTwqxd139DajXeM3jltWKtFqKsswQspG4
-tqLfKoOWhRdr3HrrSfs5FigPoY5l9Uhl9UlaoxNVFfOybW8a2wmh3Z0ouSieotCN
-OfZlE7yx1CZgL7bLkq27Z0zsZ3ZaMwrZPSbfpR9L7qecur10YzEeP71+se7uDcCU
-n4mdop6CmTu5IXZdtOU69XtFPJeYlTJ9UoH7RTSyTT4Cnj6PRmWTlbcHwfNOBejv
-UP6ekXr6trbieBXx0OuYJlrqr/3dFJ9uVr9c3P0axtoidtsvdpu3viHM2kkn9k7p
-=matT
------END PGP PUBLIC KEY BLOCK-----
\ No newline at end of file
+Ut+cDCM7zD5tcG/JTWyvw+b8ctXoceFtJFkkjHaZGBjkHpLCdQQSFgoAHRYhBIcQ
+896eXS5h2ziL7hGauPDmHZOcBQJloEdPAAoJEBGauPDmHZOcQMYA/1nM+s7LA7vu
+fL1G5cVHYR6opOVn5Vk69+pgg+ng4qMEAP98JnFPpWwoNSSc9nOSALAlgsET5E4f
+6a3Q252IPXlUBsLAXAQTAQIABgUCST/6SAAKCRBfdzkxFbItle0nB/4xQ3QOxXsO
+ZQU4805MeaLByx0MmgrDoDLY9sllytm5BmjSyeRdacofhgAqgV2mkGVFS86LPw5D
+xnXJJFLLg25WYUeS45yGDSIfL6DeELDFTUdUxdROLHGy5pv0QgnpVlUKxtBHqXrJ
+bXS1mwZfW/65Mnr+GBOiMHy5J+1UgrMgbGpmstQpX6VTsGdAHsLVupgGf1GP5Lfe
+X30zZHVXVAyNcz5Vd07wN7XmBv2ryhgYB+cevLpvegvr/JbmHIRE1oEeuSTfdddi
+LIUb34lqZxZ+AS1l3D0Ma8mpxxmWNRrBY+YHcgtgi31VkBvEr7bd+VkHk1N/8o9y
+YyupGAHfxMexzR5Sb3kgTWFycGxlcyA8cm95QG1hcnBsZXMubmFtZT7CwJQEEwEI
+AD4CGwMFCwkIBwICIgIGFQoJCAsCBBYCAwECHgcCF4AWIQSnhe0nVZVdnpPqWfZZ
+f5fqmtRVSQUCZEeJLAIZAQAKCRBZf5fqmtRVSW9iB/0WMrDBNEDWZCaZdEpMMEgJ
+4w3iA1t1J8NoAaJ7MEcze2wH0Vl9NVU4Dt61donUUFN7oUXFuJGVJ5XP6VDYrlZw
+DHjYCSQOcwi5+zyKKVOtsAyRSIme3CT7dLT0V097vQinxOgqWjH8DXz1i+8qdCiB
+9dql9Wy55w/jPdUwQA7w4eJUO47AMBAxa0twVJxmxJzLs/w/lYU8wDtM39IMGg8L
+O1X5yka53q6m9fxEC7K4CKEaFlEqWvpUARBpQ0cNUK+3+MvHoII1SUnua57rqV3B
+2bQAbhYOBOh7+PVYNBtEgluXyIbgkZkD3QjIa72ut/0r8eunCVD6aluAS8+8E124
+wsCRBBMBCAA7FiEEp4XtJ1WVXZ6T6ln2WX+X6prUVUkFAmRGnfUCGwMFCwkIBwIC
+IgIGFQoJCAsCBBYCAwECHgcCF4AACgkQWX+X6prUVUnS7wf/YNSs4SwlikJJElO9
+Vf6zCkWWyd+rnnPT/btDBs+a0dqTrVZPN+vymwGSLiDEqOnJVOma9eElUzMJWgyZ
+QJFti6LYEInnrbIzum7FPd7ofxwJ2Oh7PKaSE+n2DYBebZC4e+FW9KLWymKWAObl
+LejsTqp6jPY0OQi2de7Bi9ze4oBQiD2F9JGNcOilX4dujQbikMjTXSGvd5FGZxmA
+9y8vibPF4tqxX1lUUsJDvwL2Hb5LOaRw5cT9IAU1PcmyDtnl96sSeKc2IUgNVIMW
+D166cD+6FjjnBqCC2NRKOyAJLg9wmdewPP91QLJS80c8eLjy0iaM/2gVcJBwxZhR
+nIiT08LAkAQTAQIAIwIbAwYLCQgHAwIEFQIIAwQWAgMBAh4BAheABQJJPl+PAhkB
+ACEJEFl/l+qa1FVJFiEEp4XtJ1WVXZ6T6ln2WX+X6prUVUmEEwgAqA0FVSbmw64U
+ETH7utE/V2RTq8VmKjSwhaAq2PbZNiqqap8r/TM4x/fO15+R/UXrcfaS7AH2vSr6
+9OHMZqt6YMrmbZ7AXEVQ6bfSmPVP/ozazsR3PljyCWsgNJxFW+lqN4IpJaDyGevH
+JLTnc512sD69YRZ9JA8Tc5V95e+/qC1/ri20Dtn+FYTIeU5+1knAevhEcw73veee
+Sl99Pu0U0EqUj2r+FsEu3wQNe0nuJQHDAv//wY/R7U0gy0VKVi8MMBPJ3SZSfULH
+OdVzuizNn9PF/mO8RlOBZA1amuq+oJVCdNJLd8gmZl+3RcTJaXN9iDtipTxreRep
+kGLsJV2zp8LAjQQTAQIAIAUCSSsKggIbAwYLCQgHAwIEFQIIAwQWAgMBAh4BAheA
+ACEJEFl/l+qa1FVJFiEEp4XtJ1WVXZ6T6ln2WX+X6prUVUm6tQf/T4V6u5See/+u
+zZo6ReU7pCINQLKPETT1+RLqCYq+xnrOTguL7dzTFNu3z6qcjeu03G2WoCKrJmp5
+sFMyN0bZOhesUONdpirc5MiVSy6ddU6bPfss7AWx9/hNO3gdxuFwx4Bc0jZtpt0l
+zc1402ao+ShMn0FMTSXmCzAhO/lfQ3Cs4p/kZAI3jK7wI4Lp/xsC9sGMpOJFo6KZ
+XJlZQNxuiY35Tn9ZknXPoBWCHCuJVwaGPihkBjlpf1A/Gd/R+U/+g4vFc5mdQ71R
+UCkT7CUd/hIXNtw9YR1sz2yrf4QGj6H77zHFLLNlCuGRjQOjv2hqAs7cD5Ag+SMH
+HwHsw5co5MJ1BBIWCgAdFiEEhxDz3p5dLmHbOIvuEZq48OYdk5wFAmWgR0wACgkQ
+EZq48OYdk5xQVQEAsZ4xnFfR9M6pwXk2QODNl8BdH5eVHYCgRj5gr0AOy+sBAMko
+DS4RVar3AEeORWG7a2YFuvW7YA1sEGroQIn0zs8MwsBcBBMBAgAGBQJJLFruAAoJ
+EF93OTEVsi2V0UIH/Ar7rihCTQKPS0n4Y8LhuRpr8l/TGsoFuLpKWSiZvOguzTJ9
+yqG3VxZiEfpreEqb05JfZNYlJQPQSWOHExLOZMPIWpNtqmLd+x6djvjQDquSqaIV
+I5i/QykHhqVJrAjJJP7EzV8VIdii/zp1B0YUCXPWd6/VeuZH12fXue0a8na7Ydgx
+CGZryE1excdDHEHGUsTjWkw2VU4XZfd3/SxUuB9ddwg3+raZ/ohRTI6fjY1Tza+q
+Tk4xPRO/x9j85jLYdlVCOeL1hULpfzBwntjmJkWgjQrauPK4LPzevtMNa9Cf5Wbv
+J9OJdTWh5bbTd2w1uO+4a0quB1rGmvC5zT46HGrOwE0ESSsK+gEIAMOYyNnU8Qvu
+P68M3ODgzy3NgqwUViZCTXKiaJhpCgQWTvVYHFMN/DSCBKIWjpJnB11rVlzeRRO6
+sjOBl9wiIOCcvv20Soc2sRdysTH7K4stgLFIFXIqT9QxGeaJwe+loSwjtbrCZW7G
+90H4DMfhM/VbEZjv817434XNIAd3fn2sykOnwm9B8vMdy8jk1/vj+/lmU/YYFkHL
+Kausbkr+FHRaBLJyBBin/Po5D953BaSe/ATN2IGHSfPo0P4hqJ/G8Tg8QTeBysDk
+1UpxePY4Sas/rlkmyi/4ssn65BwsUe9W4BxOjX4XO5Pkg368CVo8/rk13uGqzqFF
+FkShj1BaRtEAEQEAAcLAdgQYAQgAIAIbDBYhBKeF7SdVlV2ek+pZ9ll/l+qa1FVJ
+BQJkRpwjAAoJEFl/l+qa1FVJ5o4H/0M8J+BaAIRcmq6IPnRknvU9FiIXCXliuPOZ
+m0a6Vc+U+a5rr8ErSJXzAK/IqtOzHn7KUrlHT/V6w5VRSmIFjR5eJVXhdcYB9Pyw
+xBjr6ffD7v2X4EGhQCTNzrbwmsXP9PF5ES4s++hZa3XO0fbfxnwOcu+4jJAENupk
+9lxduXlU2FIoKKctGpzW8RRmM8b5Afi0XLKfR8w+eL7HXtsIfYqoQMSOuCupmNd+
+iWS+RZXm0yl30qnLzKs+ZMUvWgkkTZEUFR29yYnPLBQMcq0c7QVoiCC8dtq0EwWR
+OJeo+dVtczXZORQ3VPM880fBqy2Ppu61lL8OMUlLxmrxqPFakcbCwHYEGAECAAkF
+AkkrCvoCGwwAIQkQWX+X6prUVUkWIQSnhe0nVZVdnpPqWfZZf5fqmtRVSU5lB/9N
+dyONWDwonoiRs7zNV2bf4+aUSS6G/koWHEuAlTRANWF2mzU52YrmTwqxd139DajX
+eM3jltWKtFqKsswQspG4tqLfKoOWhRdr3HrrSfs5FigPoY5l9Uhl9UlaoxNVFfOy
+bW8a2wmh3Z0ouSieotCNOfZlE7yx1CZgL7bLkq27Z0zsZ3ZaMwrZPSbfpR9L7qec
+ur10YzEeP71+se7uDcCUn4mdop6CmTu5IXZdtOU69XtFPJeYlTJ9UoH7RTSyTT4C
+nj6PRmWTlbcHwfNOBejvUP6ekXr6trbieBXx0OuYJlrqr/3dFJ9uVr9c3P0axtoi
+dtsvdpu3viHM2kkn9k7p
+=UgCe
+-----END PGP PUBLIC KEY BLOCK-----
-- 
2.55.0

